src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -112,7 +112,6 @@ const TokenSelector = styled.div<{ onClick?: () => void }>`
   align-items: center;
   justify-content: flex-end;
   width: 8.4rem;
-  height: 9.2rem;
```

Comment:
this can lead to lot of problems in mobile/safari/chrome stuff. did you test this cases?

Reply:
specially in all the modals (flex layout and stuff)

Reply:
having fixed height cuts it off on desktop 
<img width="171" alt="image" src="https://user-images.githubusercontent.com/80323528/135618876-c8c2960c-b68e-4a73-bf4f-e264e9282c19.png">

and mobile
<img width="371" alt="image" src="https://user-images.githubusercontent.com/80323528/135618935-f9e508ba-ef66-4ec0-9dab-32c31a21d36c.png">

usually as a general rule avoiding fixed heights in css allows things to move and size fluidly

Reply:
without fixed height

<img width="412" alt="image" src="https://user-images.githubusercontent.com/80323528/135619142-99b24840-be21-44a8-af35-b4e35805c697.png">

Reply:
this space is intended too, also it impact on the modals size (it changes the modal size and cause rerendering without fixed height)